all_tests.template.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Personally I prefer to keep such small scripts inline in the buildout. Inline templates are much easier to understand. Otherwise people have to look into many different files to understand what goes on.